charts/datadog/templates/_container-system-probe.yaml

```diff
@@ -78,6 +78,43 @@
       mountPath: /etc/datadog-agent/runtime-security.d
       readOnly: true
 {{- end }}
+{{- if eq (include "runtime-compilation-enabled" .) "true" }}
+    - name: runtime-compiler-output-dir
+      mountPath: {{ .Values.datadog.systemProbe.runtimeCompilationAssetDir }}/build
+      mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
+    - name: kernel-headers-download-dir
+      mountPath: {{ .Values.datadog.systemProbe.runtimeCompilationAssetDir }}/kernel-headers
+      readOnly: false
+{{- if .Values.datadog.systemProbe.mountPackageManagementDirs }}
+    - name: etc-os-release
+      mountPath: /host/etc/os-release
+      readOnly: true
+    - name: etc-redhat-release
+      mountPath: /host/etc/redhat-release
+      readOnly: true
+    - name: etc-fedora-release
+      mountPath: /host/etc/fedora-release
+      readOnly: true
+    - name: etc-lsb-release
+      mountPath: /host/etc/lsb-release
+      readOnly: true
+    - name: apt-config-dir
+      mountPath: /host/etc/apt
+      readOnly: true
+    - name: yum-repos-dir
+      mountPath: /host/etc/yum.repos.d
+      readOnly: true
+    - name: opensuse-repos-dir
+      mountPath: /host/etc/zypp
+      readOnly: true
+    - name: public-key-dir
+      mountPath: /host/etc/pki
+      readOnly: true
+    - name: rhel-subscription-dir
+      mountPath: /host/etc/rhsm
+      readOnly: true
+{{- end }}
```

Comment:
the idea that I had in my was the opposite. 
By default all the paths are added, But if the user doesn't or can't mount these path (for example read-only file system) they can provide the list of path they want.

something like:

```suggestion
{{- if not .Values.datadog.systemProbe.mountPackageManagementDirs }}
    - name: etc-os-release
      mountPath: /host/etc/os-release
      readOnly: true
    - name: etc-redhat-release
      mountPath: /host/etc/redhat-release
      readOnly: true
    - name: etc-fedora-release
      mountPath: /host/etc/fedora-release
      readOnly: true
    - name: etc-lsb-release
      mountPath: /host/etc/lsb-release
      readOnly: true
    - name: apt-config-dir
      mountPath: /host/etc/apt
      readOnly: true
    - name: yum-repos-dir
      mountPath: /host/etc/yum.repos.d
      readOnly: true
    - name: opensuse-repos-dir
      mountPath: /host/etc/zypp
      readOnly: true
    - name: public-key-dir
      mountPath: /host/etc/pki
      readOnly: true
    - name: rhel-subscription-dir
      mountPath: /host/etc/rhsm
      readOnly: true
{{- else }}
{{- range .Values.datadog.systemProbe.mountPackageManagementDirs }}
    - name: {{ .name }}
      mountPath: {{ .mountPath }}
      readOnly: true
{{- end }}
```